### DISCUSSION TITLE
multiple monitoring ports?

### DISCUSSION THREAD

**Discussion by isaacgolding** [Security-Onion-Solutions/securityonion#8132] [Unsupported Versions] [answered]
I have a situation where I don't have control over the edge of my network (SDWAN and two catalyst switches provided by the vendor)
I can request config changes and whatnot but I have no direct access to the admin side of things for the catalyst switches.   I do however have 3 more switches in my network that I do have control over and I'm considering installing an additional NIC and configuring my own SPAN (port mirror) to my forward sensor.  Only one sensor is deployed.   I have 3 VLANS, roughly 120 devices on the local lan and another 40 devices on remote lan/subnets with a vpn tunnel.   There is intra-vlan communication both local and remote.

I guess I'm asking before I tear things apart to see if this is a sound idea?   I'm assuming there will be some duplication of traffic between the two sensor inputs...and does the system deal with duplication of info?   I just don't feel that my current sensor is seeing everything I expect it to be seeing and I'm not sure how else to "easily" approach the issue.

**Comment by dougburks** [comment, marked as answer]
If you see duplication of logs, you might be able to tune using BPF:
https://docs.securityonion.net/en/2.3/bpf.html
